### PR TITLE
[Feature] Gatway Pages Refactoring

### PR DESCRIPTION
**Pull request by Civolilah**
@beganovich @turbo124 PR includes refactoring for `Payment Gateway creation` page. Here is the screenshot:

Screenshot 2023-05-24 at 02 41 27

David, if you agree with the solution, please just add missing translation keyword (`official_website` and `set_up`) in translation files.

Let me know your thoughts.

### CODE REVIEW

**Comment by Civolilah** [comment]
> In progress...

**Comment by turbo124** [comment]
image

If I delete a gateway, we don't allow the gateway to be reinstalled. ie. delete Stripe and then it isn't possible to add stripe back in.

Also, for the gateway overview list, we should add in a state column active/archived/deleted.

**Comment by Civolilah** [comment]
> image
> 
> If I delete a gateway, we don't allow the gateway to be reinstalled. ie. delete Stripe and then it isn't possible to add stripe back in.
> 
> Also, for the gateway overview list, we should add in a state column active/archived/deleted.

@turbo124 All of those things are fixed and additional that we discussed on the Slack. So, it is prepared for final review. Let me know your thoughts.

**Comment by turbo124** [comment]
When the user clicks the setup button, the gateway is created, but then they are taken to the listview, we want to redirect the user into the Edit Page of the Gateway

Also, for the link text please use the translation helper `website`

**Comment by Civolilah** [comment]
> When the user clicks the setup button, the gateway is created, but then they are taken to the listview, we want to redirect the user into the Edit Page of the Gateway
> 
> Also, for the link text please use the translation helper `website`

@turbo124 Okay, both of your suggestions are implemented. Let me know your thoughts.

File: src/pages/clients/show/components/GatewayTypeIcon.tsx [hunk 1]
```diff
@@ -23,7 +43,7 @@ export function GatewayTypeIcon(props: Props) {
         <img
           src="/gateway-card-images/visa.png"
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/pages/clients/show/components/GatewayTypeIcon.tsx]: @beganovich changed as they suggested!

**Pull request merged**